rts/Sim/Units/UnitTypes/Builder.cpp

```diff
@@ -731,11 +748,12 @@ bool CBuilder::StartBuild(BuildInfo& buildInfo, CFeature*& feature, bool& inWait
 
 				if (cu == nullptr)
 					continue;
+				if (allyteam != cu->allyteam)
```

Comment:
```suggestion
				if (!teamHandler.Ally(allyteam, cu->allyteam)
```

Reply:
If you're allied in an FFA you still don't share LoS and potentially still can't see the unit though?

Reply:
you can assist them, though, AFAIK.

Reply:
You can assist enemies too though, I've seen automexing cons assisting enemy mexes